features/overscroll-behavior.yml

```
@@ -0,0 +1,29 @@
+name: overscroll-behavior
+description: The `overscroll-behavior` CSS property sets the behavior when the edges of a scrolling area are reached. The `contain` keyword prevents scroll chaining, and the `none` keyword prevents scroll chaining, bouncing effects, and device specific behaviors like scroll to refresh.
```

It might be better to avoid references to concepts such as scroll chaining or specific boundary actions and concentrate on the intention to disable these behaviors

```suggestion
description: The `overscroll-behavior` CSS property allows disabling default scrolling behaviors when the edges of a scrolling area are reached.
```